.sonata-project.yml

```diff
@@ -24,6 +24,10 @@ labels:
   - name: help wanted
     color: '0e8a16'
 
+files:
+  - doc/CONTRIBUTING.md:CONTRIBUTING.md
+  - tools/editorconfig/.editorconfig:.editorconfig
```

Comment:
Puppet separates into a `file` folder and a `templates` directory, but I guess having different extensions for files in the same directory would be enough…